packages/lit-dev-content/site/tutorial/content/06-template-logic.md

```diff
@@ -29,6 +32,20 @@ You can use standard JavaScript in Lit expressions to create conditional or repe
     }
     ```
 
+    ```js
+    get input() {
+      return this.renderRoot?.querySelector('#newitem') ?? null;
+    }
+
+    addToDo() {
+      this.listItems.push({text: this.input.value, completed: false});
+      this.requestUpdate();
```

Comment:
while you're here, can you put `requestUpdate()` last? It being in the middle looks more significant than it is to me, while the order doesn't actually matter at all.

Reply:
Done. Also updated all the RHS project files in the same way from this step on.